pkg/image/controller/controller.go

```diff
@@ -67,29 +66,38 @@ func (c *ImportController) Next(stream *api.ImageStream) error {
 		client = dockerregistry.NewClient()
 	}
 
-	toImport, err := getTags(stream, client, insecure)
+	var errlist []error
+	toImport, retry, getErr := getTags(stream, client, insecure)
```

Comment:
Right, forgot to refactor that, I had some others ideas about merging the two errors and needed different names then.